### PR TITLE
ACC-11 initial register finctionality added, no test yet

### PR DESCRIPTION
**Pull request by mrbelpit**
ACC-11 initial regiser finctionality added, no test yet

### CODE REVIEW

**Review by szukacs — APPROVED** [review]

File: src/main/java/accenture/demo/exception/registration/EmailAddressIsAlreadyRegistered.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package accenture.demo.exception.registration;
+
+public class EmailAddressIsAlreadyRegistered extends RegistrationException {
```
Comment (spiglebach) [inline src/main/java/accenture/demo/exception/registration/EmailAddressIsAlreadyRegistered.java]: Modify class name to EmailAddressIsAlreadyRegisteredException for easier understanding

**Review by spiglebach — APPROVED** [review]
In later versions we should make package names lowercase to match java conventions.

File: src/main/java/accenture/demo/exception/registration/EmailAddressIsAlreadyRegistered.java (hunk 1 above)
Comment (mrbelpit) [reply src/main/java/accenture/demo/exception/registration/EmailAddressIsAlreadyRegistered.java]: fixed

**Comment by mrbelpit** [comment]
fixed package names, and exception name

**Pull request merged**